VERSION

```diff
@@ -1 +1 @@
-0.21.0g-beta.1
+0.21.0g-beta.2
```

Comment:
Any idea what does a letter `g` do in this version?

Reply:
@adambabik I'm not sure tbh, I just bumped the version

Reply:
maybe @mandrigin knows?